packages/utils/src/abi_encoder/abstract_data_types/types/blob.ts

```diff
@@ -0,0 +1,38 @@
+import { DataItem } from 'ethereum-types';
+import * as _ from 'lodash';
+
+import { CalldataBlock, CalldataBlocks, RawCalldata } from '../../calldata';
+import { DecodingRules } from '../../utils/rules';
+
+import { DataType } from '../data_type';
+import { DataTypeFactory } from '../interfaces';
+
+export abstract class Blob extends DataType {
```

Comment:
There are a handful of naming collisions with these classes (`Blob`, `Pointer`, `Set`). I would prefer more explicit names like `AbstractDataTypeBlob` so that we are able to import these classes directly.